tests/suite/wip_test_auth_basic_auth_mergeable.py

```diff
@@ -0,0 +1,165 @@
+import requests
```

Comment:
Is this file meant to still be here? It's marked as WIP and looks at a glance to be covering the functionality of the non-WIP named file above.